packages/icons/src/arrows/arrow-double-down.svg

```diff
@@ -1 +1 @@
-<svg id="Default" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><polygon points="20 10.75 12 16.75 4 10.75 4 12.01 12 18 20 12.01 20 10.75"/><polygon points="20 6.75 12 12.75 4 6.75 4 8.01 12 14 20 8.01 20 6.75"/></svg>
\ No newline at end of file
+<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="m18.5 12.144-6.497 5.477-.002.004h-.002l-.005-.006-6.494-5.476.642-.768 5.858 4.94 5.858-4.94z"/><path d="m18.5 9.144-6.497 5.477-.002.003-.002.001-.005-.006-6.494-5.476.642-.768 5.858 4.94 5.858-4.94z"/></svg>
```

Comment:
I only checked this one but I am wondering why the new icon is that much larger?

Old prod icon:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="100%" height="100%">
  <path d="m20 10.75-8 6-8-6v1.26L12 18l8-5.99v-1.26z"/>
  <path d="m20 6.75-8 6-8-6v1.26L12 14l8-5.99V6.75z"/>
</svg>
```

New prod icon:
```
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24">
  <path d="m18.5 12.144-6.497 5.477-.002.004h-.002l-.005-.006L5.5 12.143l.642-.768L12 16.315l5.858-4.94z"/>
  <path d="m18.5 9.144-6.497 5.477-.002.003-.002.001-.005-.006L5.5 9.143l.642-.768L12 13.315l5.858-4.94z"/>
</svg>
```

That's a 38% increase just by character count. If that's the case everywhere then something definitely isn't right.